examples/apexlog/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"errors"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	"github.com/airbrake/gobrake/v5/apexlog"
+	"github.com/apex/log"
+)
+
+func main() {
+	// Initialize the airbrake notifier
+	airbrake := gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
+		ProjectId:   363389,                             // use it as an environment variable
+		ProjectKey:  "baa755018d5e35e07897ee0087fcce9c", // use it as an environment variable
+		Environment: "production",                       // use it as an environment variable
+	})
+
+	// Close the airbrake notifier
```

Comment:
This comment doesn't look useful because it doesn't tell anything new.

Reply:
It is to generalise the example, so the people don't get confused if it requires different configuration.